modules/s3-bucket/providers.tf

```diff
@@ -26,3 +28,9 @@ variable "import_role_arn" {
   default     = null
   description = "IAM Role ARN to use when importing a resource"
 }
+
+variable "root_account_tenant_name" {
+  type        = string
+  description = "The tenant name for the root account"
+  default     = null
+}
```

Comment:
This should not be needed. `iam-roles` should be configured with this information specifically so we do not need to pass it through all the root modules.
```suggestion
```